server/vulnerabilities/nvd/cpe_matching_rules.go

```diff
@@ -0,0 +1,127 @@
+package nvd
+
+import (
+	"fmt"
+)
+
+type CPEMatchingRules []CPEMatchingRule
+
+func GetCPEMatchingRules() (CPEMatchingRules, error) {
+	// TODO: Move this to a metadata file?
+	rules := CPEMatchingRules{
+		CPEMatchingRule{
+			CPESpecs: []CPEMatchingRuleSpec{
+				{
+					Vendor:           "apple",
+					Product:          "icloud",
+					TargetSW:         "windows",
+					SemVerConstraint: "< 7.1",
+				},
+			},
+
+			CVEs: map[string]bool{
+				"CVE-2017-13797": true,
+			},
```

Comment:
Nit, for sets it's idiomatic to use `map[T]struct{}`:
```
CVEs: map[string]struct{}{
				"CVE-2017-13797": {},
}
```

Reply:
Fixed in https://github.com/fleetdm/fleet/pull/12551/commits/24040cf3cd3ac7a150c34d35068dd97f5c5f9d4d